deploy-board/deploy_board/templates/clusters/base_images_events.html

```diff
@@ -74,6 +77,25 @@ <h4 class="panel-title pull-left">Base Images Events</h4>
             },
         });
     });
+    $("#cancelBtnId").click(function () {
+        var image_id = '{{ image_id }}/';
+        $.ajax({
+            type: 'PUT',
+            url: `${location.protocol}//${location.host}/clouds/cancel_image_update/${image_id}`,
```

Comment:
Does the following work? If it does, then we don'e need line 81. 
```suggestion
            url: `${location.protocol}//${location.host}/clouds/cancel_image_update/{{ image_id }}/`,
```

Reply:
yes, make sense. ^^